frontend/src/App.tsx

```diff
@@ -111,7 +108,7 @@ export function initialAppState(cookies: Cookies): AppState {
     },
     watchGameId: null,
     isPrivate: cookies.isPrivate && cookies.isPrivate === "true" ? true : false,
-    eloId: validateEloIdOrMakeRandom(cookies.eloId),
+    idToken: "",
```

Comment:
maybe add a comment: // logged out users have an empty-string idToken

(if it's true)